src/components/PostComments.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { postedComments } from "../api";
+const PostComments = ({ article_id, comments, setComments }) => {
+  const [beenClicked, setBeenClicked] = useState(false);
+  const [commentToAdd, setCommentToAdd] = useState("");
+  const [username, setUsername] = useState("jessjelly");
+
+  useEffect(() => {
```

Comment:
The useEffect is not needed, we have the article_id, comments and setComment passed down on props. If the parent changes so will article_id. The functionality should be within handleClick which will allow you to get rid of the beenClicked state.

Reply:
hi when I take out the use effect there is no box to type comments in anymore